waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -386,8 +386,28 @@ when defined(rln) or (not defined(rln) and not defined(rlnzerokit)):
 
     return proofBytes
 
-  proc proofVerify*(rlnInstance: RLN[Bn256], data: openArray[byte],
-    proof: RateLimitProof): bool =
+  proc getMerkleRoot*(rlnInstance: RLN[Bn256]): MerkleNodeResult =
+    # read the Merkle Tree root after insertion
+    var
+      root {.noinit.}: Buffer = Buffer()
+      rootPtr = addr(root)
+      get_root_successful = get_root(rlnInstance, rootPtr)
```

Comment:
Why not `camelCase`?

Reply:
Addressed in [a0b3761](https://github.com/status-im/nwaku/pull/1158/commits/a0b376199382ddf0174a2319187d5d5399ac4f1d)